Gemfile

```diff
@@ -36,6 +36,7 @@ group :development, :test do
   gem 'factory_bot_rails'
   gem 'capybara'
   gem 'coveralls', require: false
+  gem 'devise'
```

Comment:
There is no need for the devise gem to be in the `:development, :test` group of the gems. Please clean this file up, remove the comments.